styles/styles.css

```diff
@@ -0,0 +1,861 @@
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 47px;
+  font-size: 40px;
+  color: #383838;
+  position: relative;
+}
+
+.block-two__header_a:hover::after {
+  display: block;
+  content: "";
+  height: 4px;
+  width: 100px;
+  background-color: #f5bd41;
+  position: absolute;
+  right: 33%;
+  margin-top: 20px;
+}
+
+.block-two__img {
+  background-size: 100%;
+  background-repeat: no-repeat;
+  width: 100px;
+  height: 100px;
+  justify-self: center;
+}
+
+.block-two__caption {
+  font-weight: 400;
+  font-size: 24px;
+  color: #383838;
+  padding-top: 48px;
+  justify-self: center;
+  padding-bottom: 2px;
+}
+
+.block-two__img--support {
+  grid-column: 1 / 2;
+  grid-row: 2 / 3;
+}
+
+.block-two__caption--support {
+  grid-column: 1 / 2;
+  grid-row: 3 / 4;
+}
+
+.block-two__img--communicate {
+  grid-column: 2 / 3;
+  grid-row: 2 / 3;
+}
+
+.block-two__caption--communicate {
+  grid-column: 2 / 3;
+  grid-row: 3 / 4;
+}
+
+.block-two__img--book-online {
+  grid-column: 3 / 4;
+  grid-row: 2 / 3;
+}
+
+.block-two__caption--book-online {
+  grid-column: 3 / 4;
+  grid-row: 3 / 4;
+}
+
+.block-two__img--reviews {
+  grid-column: 4 / 5;
+  grid-row: 2 / 3;
+}
+
+.block-two__caption--reviews {
+  grid-column: 4 / 5;
+  grid-row: 3 / 4;
+}
+
+@media only screen and (max-width: 360px) {
+  /* For mobile phones Block 2 */
+  .block-two__header_a:hover::after {
+    display: block;
+    content: "";
+    height: 4px;
+    width: 100px;
+    background-color: #f5bd41;
+    position: absolute;
+    right: 33%;
+    margin-top: 20px;
+  }
+
+  .block-two {
+    grid-template: auto / 214px;
+    margin: 24px 0 40px;
+    column-gap: 0;
+    justify-content: center;
+  }
+
+  .block-two__header {
+    grid-column: 1 / span 1;
+    justify-self: center;
+    padding-bottom: 32px;
+  }
+
+  .block-two__header_a {
+    line-height: 33px;
+    font-size: 28px;
+  }
+
+  .block-two__img {
+    width: 70px;
+    height: 70px;
+  }
+
+  .block-two__caption {
+    font-size: 15px;
+    padding-top: 16px;
+    padding-bottom: 32px;
+  }
+
+  .block-two__img--support {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+  }
+
+  .block-two__caption--support {
+    grid-column: 1 / 2;
+    grid-row: 3 / 4;
+  }
+
+  .block-two__img--communicate {
+    grid-column: 1 / 2;
+    grid-row: 4 / 5;
+  }
+
+  .block-two__caption--communicate {
+    grid-column: 1 / 2;
+    grid-row: 5 / 6;
+  }
+
+  .block-two__img--book-online {
+    grid-column: 1 / 2;
+    grid-row: 6 / 7;
+  }
+
+  .block-two__caption--book-online {
+    grid-column: 1 / 2;
+    grid-row: 7 / 8;
+  }
+
+  .block-two__img--reviews {
+    grid-column: 1 / 2;
+    grid-row: 8 / 9;
+  }
+
+  .block-two__caption--reviews {
+    grid-column: 1 / 2;
+    grid-row: 9 / 10;
+    padding-bottom: 0;
+  }
+}
+
+/*for Block 3 */
+.block-three__background {
+  background-color: #eaf0f9;
+}
+
+.block-three {
+  display: grid;
+  grid-template: auto / repeat(4, 25%);
```

Comment:
The content is out of the grid-container here. Use 1fr instead of %.

Reply:
ok, thank you.